ghost/core/core/server/services/email-service/email-templates/partials/styles.hbs

```diff
@@ -335,6 +349,48 @@ h6 + .kg-paywall .kg-paywall-hr td {
     {{/if}}
 }
 
+{{#hasFeature "emailCustomizationAlpha"}}
+/* Apply custom content text color to main content elements */
+.post-content-row > td > p,
+.post-content-row .post-content > p,
+.post-content-row .post-content-sans-serif > p,
+.post-content-row > td > blockquote,
+.post-content-row .post-content > blockquote,
+.post-content-row .post-content-sans-serif > blockquote,
+.post-content-row > td > blockquote p,
+.post-content-row .post-content > blockquote p,
+.post-content-row .post-content-sans-serif > blockquote p,
+.post-content-row .kg-cta-bg-none .kg-cta-text p {
+    color: {{#if backgroundIsDark}}#ffffff{{else}}#15212A{{/if}} !important;
+}
+
+/* Exclude paragraphs inside cards from custom text color (except CTA cards which are handled separately) */
+.post-content-row .kg-card:not(.kg-cta-card) p,
+.post-content-row .kg-toggle-content p,
+.post-content-row .kg-callout-text,
+.post-content-row .kg-product-description-wrapper p {
+    color: inherit !important;
+}
+
+/* Exclude CTA cards with colored backgrounds from custom text color, but allow transparent ones */
+.post-content-row .kg-cta-bg-white .kg-cta-text p,
+.post-content-row .kg-cta-bg-grey .kg-cta-text p,
+.post-content-row .kg-cta-bg-blue .kg-cta-text p,
+.post-content-row .kg-cta-bg-green .kg-cta-text p,
+.post-content-row .kg-cta-bg-yellow .kg-cta-text p,
+.post-content-row .kg-cta-bg-red .kg-cta-text p,
+.post-content-row .kg-cta-bg-pink .kg-cta-text p,
+.post-content-row .kg-cta-bg-purple .kg-cta-text p {
```

Comment:
I wonder if we could simplify this with `.post-content-row [class~="kg-cta-bg"] .kg-cta-text p {` 🤔 

Juice should inline properly still I think so we don't lose Outlook support